docs/dev-proc/versioning.md

```diff
@@ -14,66 +14,93 @@ in menu on the left side (Supported Hardware->Hardware Model->Releases).
 
 Dasharo Releases can be divided into two categories:
 
-* Dasharo Entry Subscription Releases (previous Dasharo Supporters Release for
-  Dasharo Support Entrance Subscribers)
-* Dasharo Community Releases
-
-## Dasharo Entry Subscription Releases
-
-Dasharo Entry Subscription subscribers receive firmware updates more
-frequently than the community. The number of updates per year depends on the
-number of Dasharo Entry Subscriptions sold and the availability of other
-funding (e.g., NLNet, corporate sponsors, community donations) but is less
-than 2 updates per year. Dasharo Entry Subscription Releases are characterized
-by a changing patch version (`z`). Fixes and features introduced in Dasharo
-Entry Subscription Releases will also be available later as Dasharo Community
-Releases with public pre-built binaries in the respective release pages. In
-short, being a Dasharo Subscriber gives early access to the newest features
-and fixes.
+- Dasharo Pro/Enterprise Package (formerly Dasharo Entry Subscription) Releases
+- Dasharo Community Releases
+
+## Dasharo Pro/Enterprise Package Releases
```

Comment:
To not break other references/mentions of DES, maybe we should grep over the repo and change accordingly?

Reply:
@miczyg1 yes, we should. I cannot do it right now, so it would have to wait for better times.

Reply:
@miczyg1 most of the instances were already changed by other commits. grep returns following:

```sh
[1:27:04] pietrushnic:docs git:(use-new-naming-scheme-for-des) $ grep "Dasharo Entry Subscription" **/*.md -r
docs/dev-proc/versioning.md:- Dasharo Pro/Enterprise Package (formerly Dasharo Entry Subscription) Releases
docs/dev-proc/versioning.md:Dasharo Pro/Enterprise Package (formerly Dasharo Entry Subscription)
docs/dev-proc/versioning.md:Package (formerly Dasharo Entry Subscription) sold and the availability of
docs/dev-proc/versioning.md:Dasharo Entry Subscription) Releases will also be available later as Dasharo
```

All those instances are part of `formerly` statements which are ok.

Reply:
We had `Dasharo Pro Package Releases` and now change `Dasharo Pro/Enterprise Package Releases`. It is still a change that may break something, which we may have predicted in the other commits with renaming.

Is Dasharo Pro Package and Dasharo Enterprise package the same? If yes, then why we have different names? If not, why don't we describe it in separate section (that tway we also avoid breaking other possible links to `Dasharo Pro Package Releases` section)?

Reply:
> We had Dasharo Pro Package Releases and now change Dasharo Pro/Enterprise Package Releases. It is still a change that may break something, which we may have predicted in the other commits with renaming.

Ok, I can simplify that to Dasharo Pro Package.

> Is Dasharo Pro Package and Dasharo Enterprise package the same? If yes, then why we have different names? If not, why don't we describe it in separate section (that tway we also avoid breaking other possible links to Dasharo Pro Package Releases section)?

https://docs.dasharo.com/dasharo-naming-convention/

Reply:
https://github.com/Dasharo/docs/pull/841/commits/937e5c07872200f133573c328262d771b179d8d0#diff-2fbf769fce1a51e7078b34a3c1fcfcb3c5b376ae925ba1c0d5954aaecb0ed34cL20-R20